creator-node/src/tracer.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  trace,
+  diag,
+  DiagConsoleLogger,
+  DiagLogLevel
+} from '@opentelemetry/api'
+
+import { registerInstrumentations } from '@opentelemetry/instrumentation'
+import { NodeTracerProvider } from '@opentelemetry/sdk-trace-node'
+import { Resource } from '@opentelemetry/resources'
+import {
+  // SemanticAttributes,
+  SemanticResourceAttributes as ResourceAttributesSC
+} from '@opentelemetry/semantic-conventions'
+import { RedisInstrumentation } from '@opentelemetry/instrumentation-redis'
+import { ExpressInstrumentation } from '@opentelemetry/instrumentation-express'
+import { HttpInstrumentation } from '@opentelemetry/instrumentation-http'
+import { BunyanInstrumentation } from '@opentelemetry/instrumentation-bunyan'
+import { PgInstrumentation } from '@opentelemetry/instrumentation-pg'
+
+// Not functionally required but gives some insight what happens behind the scenes
+diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.INFO)
+
+const SERVICE_NAME = 'content-node'
+
+export const setupTracing = () => {
+  const provider = new NodeTracerProvider({
+    resource: new Resource({
+      [ResourceAttributesSC.SERVICE_NAME]: SERVICE_NAME
+    })
+  })
+  registerInstrumentations({
+    tracerProvider: provider,
+    instrumentations: [
+      // Express instrumentation expects HTTP layer to be instrumented
+      new HttpInstrumentation(),
+      new ExpressInstrumentation(),
+      new RedisInstrumentation(),
+      new PgInstrumentation(),
+      new BunyanInstrumentation({
```

Comment:
so this gives us trace data in bunyan logs with no code changes required to each log call right?

Reply:
Yes - the caveat is that we have to add traces to the code first otherwise it won't add anything. 

`getTracer().startActiveSpan('foo', span => { logger.info('bar') }) `

^ This would add the trace data and context to the bunyan log

Reply:
oh ok, so my assumption was wrong - we would have to explicitly modify every single log call
there should be some abstraction possible here right? def want to avoid modifying each log call

Reply:
Well we just have to make sure the log statement is inside a span, this will also work

```js
function one() {
  getTracer().startActiveSpan('foo', span => {
      // do some work
      two()
     // do more work
    span.end()
  })
}

function two() {
    // do some work
    three()
   // do more work
}

function three() {
    logger.info('foo')
}
```

Reply:
The `ExpressInstrumentation` adds spans to every route so that already covers the majority of the log statements

Reply:
ok, i think i follow